lib/mkdev_hanami/services/intercom.rb

```diff
@@ -0,0 +1,36 @@
+require 'intercom'
+
+module Services
+  class Intercom
```

Comment:
Is it recommended way to implement service objects? Ain't Hanami Entities better choice?

Reply:
Also: https://github.com/hanami/utils

Reply:
Also search for interactors here https://discourse.hanamirb.org/t/hanami-2-0-ideas/306/2

Reply:
Also: https://speakerdeck.com/gizotti/richer-service-objects-with-hanami-interactors